### PR TITLE
Add example that reads a local file, writes to a DFS path provided by th...

### PR DESCRIPTION
**Pull request by rnowling**
...e user, reads the file back from the DFS, and compares word counts on the local and DFS versions. Useful for verifying DFS correctness.

### CODE REVIEW

**Comment by rnowling** [comment]
@andrewor14 @pwendell Happy New Year!  Ping on this PR?  Thanks!

**Comment by srowen** [comment]
@andrewor14 if you like the change I think you can merge it; no other comments here. Otherwise i think this has timed out.

File: examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+      .flatMap(_.split("\t"))
+      .filter(_.size > 0)
+      .map(w => (w, 1))
+      .countByKey()
+      .values
+      .sum
+
+    sc.stop()
+
+    if(localWordCount == dfsWordCount) {
```
Comment (andrewor14) [inline examples/src/main/scala/org/apache/spark/examples/DFSReadWriteTest.scala]: style: space after `if`

**Comment by andrewor14** [comment]
@rnowling sorry for slipping on this. I'll merge this into master after addressing the latest comments myself. Thanks for your patience.

**Comment by rnowling** [comment]
Thanks @andrewor14 !

**Pull request closed without merging**